src/App.css

```diff
@@ -174,7 +203,12 @@
 }
 
 .trendall:focus {
-  border: 2px #F4EEE0 solid;
+  border: 2px #b73840 solid;
+}
+
+
+header {
+  color: #F4EEE0;
```

Comment:
The responsive design on this app is a little strange. In larger views all of the content is squeezed in to the first third of the screen with the other 2 thirds empty. Also the comments section is very easy to break by squeezing screen size. I think you should play with the responsiveness to check all screen sizes work